src/libsync/syncjournaldb.cpp

```diff
@@ -80,12 +84,36 @@ bool SyncJournalDb::maybeMigrateDb(const QString& localPath, const QString& abso
             return false;
         }
     }
+    if( FileSystem::fileExists( newDbNameWal ) ) {
+        if( !FileSystem::remove(newDbNameWal, &error) ) {
+            qDebug() << "Database migration: Could not remove db WAL file" << newDbNameWal
+                     << "due to" << error;
+            return false;
```

Comment:
It's not fatal but would keep a stale file there. I don't know, i don't have a further opinion